site/src/contexts/auth/RequireAuth.tsx

```diff
@@ -67,7 +67,7 @@ export const RequireAuth: FC = () => {
 };
 
 type RequireKeys<T, R extends keyof T> = Omit<T, R> & {
-  [K in keyof Pick<T, R>]: NonNullable<T[K]>;
+  [K in keyof Pick<T, R>]-?: NonNullable<T[K]>;
```

Comment:
What `-?` does?

Reply:
"make the mapped key non-optional". it's weird but `NonNullable` is not enough on it's own. 🙃